Makefile

```diff
@@ -43,8 +43,6 @@ SECRET_ACCESS_KEY_FILE := .kube-secrets/aws/secret_access_key.secret
 #########################################
 
 TOOLS_DIR := hack/tools
-# temporarily until Gardener version has been revendored >= v1.54
-GOLANGCI_LINT_VERSION := v1.49.0
```

Comment:
Meanwhile I think that the newest vendored version > 1.50 and we wouldn't need this.

Reply:
It's been removed here. What's your point?